src/modules/apps/install.ts

```diff
@@ -116,13 +93,12 @@ const prepareInstall = async (appsList: string[], force: boolean): Promise<void>
         log.error(e.response.data.message)
       } else {
         switch (e.message) {
-          case InstallStatus.USER_HAS_NO_BUY_APP_LICENSE:
-            log.error(
-              `You do not have permission to purchase apps. Please check your VTEX IO 'Buy Apps' resource access in Account Managament`
-            )
+          case 'app_store_contract_not_found':
```

Comment:
For the sake of consistency. I would recommend creating a constant for it in the `InstallStatus` variable.

Reply:
Done! https://github.com/vtex/toolbelt/pull/1092/commits/493777f80d032b6c1b4b00fe28dc9768070072c9